test/checkHotNet2Consensus.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+# Setup directories.
+hotnet2=../
+hotnet2_test=/tmp/hotnet2_consensus_test
+data=$hotnet2_test/data
+results=$hotnet2_test/results
+
+# Setup parameters.
+beta=0.4
+num_network_permutations=100
+num_heat_permutations=200
+num_cores=8
+
+mkdir -p $data
+mkdir -p $results
+
+for i in 1 2 3
+do
+    mkdir -p $data/networks/$i
+    mkdir -p $data/networks/$i/permuted
+done
+
+for j in 1 2
+do
+    mkdir -p $data/weights/$j
+done
+
+# Create networks and vertex weights.
+echo Creating networks and vertex weights...
+python $hotnet2/test/createSimulatedData.py \
+    -n 250 \
+    -m 750 \
```

Comment:
Seems like the number of nodes and min number of edges should be parameters as well?